CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 1.55.0
+* Feature - Introduce jitter for task cleanup wait duration. [#2969](https://github.com/aws/amazon-ecs-agent/pull/2969)
```

Comment:
can you use the changelog in the "Description for the changelog" in pr description - https://github.com/aws/amazon-ecs-agent/pull/2969

Reply:
Sure, lemme get a new PR for this